src/components/button/Button.js

```diff
@@ -29,7 +29,7 @@ const Button = (props) => {
         styles[typography] || styles.uppercase,
       )}
     >
-      {children}
+      {children}{' '}
```

Comment:
Is the extra `{' '}` necessary?

Reply:
ugh, thats my linter doing that. :( Want me to remove it and resubmit?

Reply:
yes please

Reply:
+1. Gave the approval but please, remove this! Good job :+1:

Reply:
removed :)